modules/creating-a-machine-pool-ocm.adoc

```diff
@@ -147,7 +147,20 @@ Your Amazon EC2 Spot Instances might be interrupted at any time. Use Amazon EC2
 If you select *Use Amazon EC2 Spot Instances* for a machine pool, you cannot disable the option after the machine pool is created.
 ====
 endif::openshift-rosa-hcp[]
-
+ifdef::openshift-dedicated[]
```

Comment:
I think it would help to add additional notes here to make it clear that -  by default - instances in the machinepools inherit the shieldedVM setting from cluster level setting but a user can override the cluster level setting  by it setting it at the machinepool level